src/go/rpk/pkg/cli/cmd/cluster/license/info.go

```diff
@@ -37,16 +38,23 @@ func newInfoCommand(fs afero.Fs) *cobra.Command {
 			out.MaybeDie(err, "unable to retrieve license info: %v", err)
 
 			if !info.Loaded {
-				out.Die("this cluster is missing a license")
+				if format == "json" {
+					out.Die("{}")
```

Comment:
This could be `{"error": "this cluster is missing a license"}

Reply:
@graphcareful good point ^